database/migrations/2017_03_24_014813_CreateGroupExpensesTable.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateGroupExpensesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('groupexpenses', function (Blueprint $table) {
+            $table->increments('id');
+            $table->unsignedInteger('group_id');
+            $table->unsignedInteger('amount');
```

Comment:
amount should not be an unsigned integer. float or decimal or double or i dont know